### ISSUE TITLE
On the problem of using cityscapes dataset to train the segmentation model

### ISSUE THREAD

**Issue by tensorflowt** [Dawson-huang/Pytorch-ENet-Nice#1]
Hello! I have a curious question. Why not use color image labels instead of black and white image labels in the segmentation sample?
![aachen_000000_000019_gtFine_color](https://user-images.githubusercontent.com/37217594/72047768-3f166f80-32f6-11ea-8410-75c051c0ce90.png)
![aachen_000000_000019_gtFine_labelTrainIds](https://user-images.githubusercontent.com/37217594/72047794-4a699b00-32f6-11ea-82be-ae044290c6b9.png)
